public/schneiderpuppe.svg

```diff
@@ -0,0 +1,17 @@
+  <g id="Gruppe_21" data-name="Gruppe 21" transform="translate(1.294 7.298)">
```

Comment:
Some of these properties are export artifacts. You can remove them (`id`, `data-name`).
It's possible to automatize this by using SVG cleaner/minifier tools.